js/filter-panel.js

```diff
@@ -5,9 +5,11 @@ var _ = require('underscore');
 
 var FilterSet = require('./filter-set').FilterSet;
 var accessibility = require('./accessibility');
+var filterTags = require('./filter-tags');
```

Comment:
Good catch, Hound! Not sure why we got three of these messages. Anyway, we should probably make this change and verify that any lingering unused code is gone @noahmanger.